java/test/jmri/jmrix/SystemConnectionMemoTest.java

```diff
@@ -14,36 +14,20 @@
  */
 public class SystemConnectionMemoTest {
 
+    SystemConnectionMemo t = null;
+
     @Test
     public void testCTor() {
-        SystemConnectionMemo t = new SystemConnectionMemo("T", "Test") {
-            @Override
-            protected java.util.ResourceBundle getActionModelResourceBundle() {
-                return null;
-            }
-        };
         Assert.assertNotNull("exists", t);
     }
 
     @Test
     public void testGetConsistManagerNull() {
-        SystemConnectionMemo t = new SystemConnectionMemo("T", "Test") {
-            @Override
-            protected java.util.ResourceBundle getActionModelResourceBundle() {
-                return null;
-            }
-        };
         Assert.assertNull("null consist manager", t.get(jmri.ConsistManager.class));
     }
 
     @Test
     public void testProvidesConsistManagerNull() {
-        SystemConnectionMemo t = new SystemConnectionMemo("T", "Test") {
-            @Override
-            protected java.util.ResourceBundle getActionModelResourceBundle() {
-                return null;
-            }
-        };
```

Comment:
Was this test code removed because it was failing? Empty tests aren't particularly useful, so it would be good to put this back.

Reply:
This common block was moved to the ``setUp()`` method